Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -1154,14 +1154,28 @@ public class GraphTraverser: GraphTraversing {
             .first(where: { $0.target.product.canHostTests() })?.graphTarget
     }
 
-    func canEmbedProducts(target: Target) -> Bool {
+    func canEmbedBinaries(target: Target) -> Bool {
```

Comment:
By binary, we also mean frameworks, right? I'd add a comment what we mean by binaries here.